lib/request_handler/fieldsets_parser.rb

```diff
@@ -29,13 +29,18 @@ def run
     private
 
     def parse_options(type, values)
+      raise ExternalArgumentError, fieldsets: "invalid type: <#{type}>" if allowed[type] == false
```

Comment:
as config allow `true`, `false` or an enum

but when validating `false` is treated the same way as if there was no config at all. See the two example configs above. both of them should be valid and work the same.